azure-spring-boot-bom/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-spring-boot-bom</artifactId>
-    <version>0.2.4-SNAPSHOT</version>
+    <version>2.0.1-SNAPSHOT</version>
```

Comment:
version in readme files need to be updated also.

Reply:
@yungez Updated.